dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerAPIImpl.java

```diff
@@ -719,16 +799,35 @@ public Host getParentHost(final Container cont, final User user, final boolean r
 		return hostAPI.findParentHost(cont, user, respectFrontendRoles);
 	}
 
-	@CloseDBIfOpened
+	@Deprecated

```

Comment:
if deprecated, just in case mention on the doc, which one is the new method

Reply:
It is already mentioned in the Javadoc in the `ContainerAPI` class.